tests/queries/0_stateless/01284_fuzz_bits.sql

```diff
@@ -0,0 +1,4 @@
+SELECT fuzzBits(toString('string'), 1); -- { serverError 43 }
+SELECT fuzzBits('', 0.3); -- { serverError 44 }
```

Comment:
It would be interesting, although maybe not strictly necessary, to add a test that checks that we got the expected probability of bit flips, by fuzzing e.g. a zero fixed string and calculating the distribution of the number of one bytes.

Reply:
Done